lib/Dialect/HW/CMakeLists.txt

```diff
@@ -10,13 +10,13 @@ add_circt_dialect_library(CIRCTHW
 
   DEPENDS
   MLIRHWIncGen
+  MLIRHWAttrIncGen
+  MLIRHWStructsIncGen
```

Comment:
MLIRHWStructsIncGen can this be dropped?

Reply:
Yup and dropped!